Flank/flank
# feat: Added Flank Wrapper

Fixes #2041

This is the solution to have evergreen Flank. The wrapper will always use the latest version of Flank

## Test Plan
> How do we know the code works?

1. Build Flank wrapper with `./gradlew flank_wrapper:shadowJar`
2. Run it with `flankw/flankw.bat` or `java -jar flank_wrapper.jar` and provide flank options as usual
3. Please notice that the latest version should be downloaded just once.

## Checklist

- [x] Flank wrapper created
- [x] Flank wrapper release workflow prepared
- [x] SHA-256 sum added to Flank
